puppetserver/docker-entrypoint.d/99-log-config.sh

```diff
@@ -12,8 +12,11 @@ if [ -n "${CERTNAME}" ]; then
   echo "* CERTNAME: '${CERTNAME}'"
   certname=${CERTNAME}.pem
 else
-  echo "* CERTNAME: unset"
-  certname=$(cd "${SSLDIR}/certs" && ls *.pem | grep --invert-match ca.pem)
+  echo "* CERTNAME: unset,  try to use the oldest certificate in the certs directory, because this might be the one that was used initially."
```

Comment:
```suggestion
  echo "* CERTNAME: unset, try to use the oldest certificate in the certs directory, because this might be the one that was used initially."
```